src/tools/bump-stage0/src/main.rs

```diff
@@ -6,7 +6,7 @@ use std::convert::TryInto;
 
 const PATH: &str = "src/stage0.json";
 const COMPILER_COMPONENTS: &[&str] = &["rustc", "rust-std", "cargo"];
-const RUSTFMT_COMPONENTS: &[&str] = &["rustfmt-preview"];
+const RUSTFMT_COMPONENTS: &[&str] = &["rustfmt-preview", "rustc", "rust-std"];
```

Comment:
I think only rustc is necessary. According to the manifest it contains both lib/librustc_driver-*.so and lib/libstd-*.so. The rust-std component stores it's libstd-*.so in lib/rustlib/<triple>/lib.

Reply:
Ah, having both was my suggestion, but I think you're right.